tests/dat/actions/loggingTimeout.js

```diff
@@ -0,0 +1,35 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more contributor
+// license agreements; and to You under the Apache License, Version 2.0.
+
+// This action prints log lines to stdout for the duration of durationMillis.
+// The output is throttled by a delay of delayMillis between two log lines in
+// order to keep the log small and to stay within the log size limit.
+
+function doLog(startMillis, durationMillis, delayMillis) {
+  var lastLog = 0;
+  var logLines = 0;
+  while (true) {
+     now = new Date();
+     if (now - lastLog > delayMillis) {
```

Comment:
What happens if no parameter named `delayMillis` is passed? I guess the value is `undefined`?

Reply:
The nodejs:6 action now provides default in case of missing arguments. We do not added type checking as this is just test case and we wanted to keep the code simple.